File: conformance-tests/jest-globals/implicit-globals/jest-test-implicit-globals-with-variants_m7.js
describe('normal describe test', () => {
  it('normal it in describe test', () => {});
  test('normal test in describe test', () => {});
  fit('fit in describe test', () => {});
  xit('xit in describe test', () => {});
  xtest('xtest in describe test', () => {});
});
fdescribe('fdescribe test', () => {
  fit('fit in fdescribe test', () => {});
  xit('xit in fdescribe test', () => {});
  xtest('xtest in fdescribe test', () => {});
});
xdescribe('xdescribe test', () => {
  fit('fit inside xdescribe test', () => {});
  xit('xit inside xdescribe test', () => {});
  xtest('xtest inside xdescribe test', () => {});
});

File: conformance-tests/jest-globals/implicit-globals/jest-test-implicit-globals-with-variants_m7.lua
-- ROBLOX upstream: https://github.com/Roblox/js-to-lua/blob/sha/conformance-tests/jest-globals/implicit-globals/jest-test-implicit-globals-with-variants_m7.js
local Packages --[[ ROBLOX comment: must define Packages module ]]
local JestGlobals = require(Packages.Dev.JestGlobals)
local describe = JestGlobals.describe
local fdescribe = JestGlobals.fdescribe
local fit = JestGlobals.fit
local it = JestGlobals.it
local test = JestGlobals.test
local xdescribe = JestGlobals.xdescribe
local xit = JestGlobals.xit
local xtest = JestGlobals.xtest
describe("normal describe test", function()
	it("normal it in describe test", function() end)
	test("normal test in describe test", function() end)
	fit("fit in describe test", function() end)
	xit("xit in describe test", function() end)
	xtest("xtest in describe test", function() end)
end)
fdescribe("fdescribe test", function()
	fit("fit in fdescribe test", function() end)
	xit("xit in fdescribe test", function() end)
	xtest("xtest in fdescribe test", function() end)
end)
xdescribe("xdescribe test", function()
	fit("fit inside xdescribe test", function() end)
	xit("xit inside xdescribe test", function() end)
	xtest("xtest inside xdescribe test", function() end)
end)
